modules/xmpp/JingleSessionPC.js

```diff
@@ -1701,28 +1716,6 @@ export default class JingleSessionPC extends JingleSession {
      * @private
      */
     _initiatorRenegotiate(remoteDescription) {
```

Comment:
I have to prove this. But you can merge, because we not going live with this state! Firefox can also be the offerer, i.e., it can create an offer and send it to another client for establishing a peer-to-peer connection with this changes from jallamsetty